containerserver/server.go

```diff
@@ -272,7 +272,11 @@ func (server *ContainerServer) ContainerPutHandler(writer http.ResponseWriter, r
 	}
 	created, db, err := server.containerEngine.Create(vars, timestamp, metadata, policyIndex, defaultPolicyIndex)
 	if err == ErrorPolicyConflict {
-		srv.StandardResponse(writer, http.StatusConflict)
+		if policyIndex == -1 {
```

Comment:
Ah, that's quite probable. I started here and then ended up in directclient when fixing that issue. I'll remove this part and run it through again. Thanks